model/src/pyrenew/latent/infections.py

```diff
@@ -25,18 +19,18 @@ class Infections(RandomVariable):
 
     def __init__(
         self,
-        gen_int: ArrayLike,
+        gen_int_varname: str = "gen_int",
         I0_varname: str = "I0",
         Rt_varname: str = "Rt",
         infections_mean_varname: str = "latent_infections",
-        I0_dist: dist.Distribution = dist.LogNormal(2, 0.25),
     ) -> None:
         """Default constructor
 
         Parameters
         ----------
-        gen_int: ArrayLike
-            A vector representing the pmf of the generation interval
+        gen_int_varname : str.
+            Name of the element in `random_variables` that will hold the value
+            of 'generation interval'.
```

Comment:
I think that if we find ourselves doing things like this it's a good argument for the `sample(named_arg1, named_arg2, **kwargs)` pattern instead.

Reply:
Agreed. This will involve a no minor lift, so I'll create a separate issue for this.

Reply:
Furthermore, we should talk about this tomorrow. I'll put it on the agenda.